content/zh/docs/ops/best-practices/security/index.md

```diff
@@ -7,19 +7,409 @@ owner: istio/wg-security-maintainers
+
+### 如何配置{#how-to-configure}
+
+您可以使用 `istioctl` 命令来更新[网格配置](/zh/docs/reference/config/istio.mesh.v1alpha1/):
+
+{{< text bash >}}
+$ istioctl upgrade --set meshConfig.pathNormalization.normalization=DECODE_AND_MERGE_SLASHES
+{{< /text >}}
+
+或者修改您的 operator 来复写文件
+
+{{< text bash >}}
+$ cat <<EOF > iop.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  meshConfig:
+    pathNormalization:
+      normalization: DECODE_AND_MERGE_SLASHES
+EOF
+$ istioctl install -f iop.yaml
+{{< /text >}}
+
+或者，如果您想直接修改网格配置，您可以将 [`pathNormalization`](/zh/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig-ProxyPathNormalization) 加入[网格配置](/zh/docs/reference/config/istio.mesh.v1alpha1/)中，即修改 `istio-system` 命名空间中的 `istio-<REVISION_ID>` configmap。
+例如，如果您使用 `DECODE_AND_MERGE_SLASHES` 选项，您需要按照如下修改网格配置：
+
+{{< text yaml >}}
+apiVersion: v1
+  data:
+    mesh: |-
+      ...
+      pathNormalization:
+        normalization: DECODE_AND_MERGE_SLASHES
+      ...
+{{< /text >}}
+
+### 较不常见规范化配置{#less-common-normalization-configurations}
+
+#### 大小写规范化{#case-normalization}
+
+在一些环境下，需要不区分授权策略中路径的大小写。
+例如，将 `https://myurl/get` 和 `https://myurl/GeT` 视为等效。
+在这些情况下，可以使用如下的 `EnvoyFilter`。
+此设置将同时改变策略对比所用的以及传递给应用的路径。
+
+{{< text syntax=yaml snip_id=ingress_case_insensitive_envoy_filter >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: ingress-case-insensitive
+  namespace: istio-system
+spec:
+  configPatches:
+  - applyTo: HTTP_FILTER
+    match:
+      context: GATEWAY
+      listener:
+        filterChain:
+          filter:
+            name: "envoy.filters.network.http_connection_manager"
+            subFilter:
+              name: "envoy.filters.http.router"
+    patch:
+      operation: INSERT_BEFORE
+      value:
+        name: envoy.lua
+        typed_config:
+            "@type": "type.googleapis.com/envoy.extensions.filters.http.lua.v3.Lua"
+            inlineCode: |
+              function envoy_on_request(request_handle)
+                local path = request_handle:headers():get(":path")
+                request_handle:headers():replace(":path", string.lower(path))
+              end
+{{< /text >}}
+
+## 理解流量拦截的局限性{#understand-traffic-capture-limitations}
+
+Istio sidecar 原理为拦截入站和出站流量并将它们转发到 sidecar 代理。
+
+但是，并不是*全部*的流量都被拦截：
+
+* 转发只针对基于 TCP 的流量。任何 UDP 或 ICMP 包不会被拦截或更改。
+* 入站拦截在很多 [sidecar 使用的端口](/zh/docs/ops/deployment/requirements/#ports-used-by-istio) 以及端口 22 不生效。此列表可以通过例如 `traffic.sidecar.istio.io/excludeInboundPorts` 的设置拓展。
+* 出站拦截可以通过类似 `traffic.sidecar.istio.io/excludeOutboundPorts` 的配置以及其他多种方式取消。
+
+总的来说，一个应用和他的 sidecar 代理之间的安全边界微乎其微。Sidecar 设置基于 pod 粒度设置，并且二者运行在同一个网络/进程空间。因此，应用可能移除拦截规则，并且移除，修改，或替换 sidecar 代理。这允许一个 pod 将出站流量有意地绕过它的 sidecar 或者 有意允许入站流量绕过它的 sidecar。
+
+因此，仅依赖 Istio 来拦截全部流量是不安全的。取而代之，正确的安全边界是，一个客户端不应该能够绕过*另一个* pod 的 sidecar。
+
+例如，如果在端口 `9080` 上运行 `reviews` 应用，便应认为所有从 `productpage` 应用来的流量都应被 `review` sidecar 代理拦截，在 sidecar 上便可以进行 Istio 认证和授权策略配置。
+
```

Comment:
I suggest Sidecar instead of sidecar and Pod instead of pod

Reply:
thanks for the suggestions, all fixed